astropy/stats/lombscargle/implementations/tests/test_lombscargle.py

```diff
@@ -0,0 +1,139 @@
+import pytest
```

Comment:
I think the preference is to use the pytest bundled with astropy for the testing, so this would be:

`from ......tests.helper import pytest`

It is highly likely I've miscounted the number of `.`s needed...